docs/raw/push-notification-server.md

```diff
@@ -297,17 +294,16 @@ message PushNotificationQuery {
 }
 ```
 
-MUST be sent to the server. 
+MUST be sent to the server.
 
-The server MUST be checking the signature of the message and comparing
-that with the latest `PushNotificationPreferences` for the given public key.
-
-If the requester has access to any access token, a response MUST be sent:
+A response MUST be sent:
 
 ```protobuf
 message PushNotificationQueryInfo {
   string access_token = 1;
   string installation_id = 2;
+  bytes public_key = 3;
```

Comment:
That's a bit tricky,
I'd say, let's park the renaming for now and we will do in a different PR if it that's  ok with you.
The thing I am not sure is that not always this will be chat keys, for example in anonymous mode this are just keys that a user creates specifically for push notifications, but they are not chat keys.

Reply:
ok, so I have updated the references where we refer 100% to a chat key